libs/SmartStore/SmartStore/Classes/SFSmartStore.m

```diff
@@ -861,14 +861,21 @@ - (NSString*)loadExternalSoupEntryAsString:(NSNumber *)soupEntryId
 }
 
 - (NSString*) readFromEncryptedFile:(NSString*)filePath
-                                key:(NSData*)key
-                                 iv:(NSData*)iv
+                             encKey:(SFEncryptionKey*)encKey
+{
+    return [self readFromEncryptedFile:filePath encKey:encKey useNilIV:NO];
+}
+
+- (NSString*) readFromEncryptedFile:(NSString*)filePath
+                             encKey:(SFEncryptionKey*)encKey
+                           useNilIV:(BOOL)useNilIV
 {
     NSMutableString* content = [NSMutableString new];
     NSInputStream *inputStream = nil;
-    if (key) {
+    if (encKey) {
         SFDecryptStream *decryptStream = [[SFDecryptStream alloc] initWithFileAtPath:filePath];
```

Comment:
I could go further and change SFDecryptStream to work with SFEncryptionKey instead of its components.